[10.0] Update invoice2data library reference link

matteotrubini opened the pull request:
From https://github.com/m3nu/invoice2data to https://github.com/invoice-x/invoice2data

alexis-via commented on account_invoice_import_invoice2data/README.rst:
```diff
@@ -60,11 +60,11 @@ Go to the form view of the supplier and configure it with the following paramete
 * in the *Accounting* tab, create an *Invoice Import Configuration*.
 
-For the PDF invoice of your supplier that don't have an embedded XML file, you will have to create a `template file <https://github.com/m3nu/invoice2data/blob/master/invoice2data/templates>`_ in YAML format in the invoice2data Python library. It is quite easy to do ; if you are familiar with `regexp <https://docs.python.org/2/library/re.html>`_, it should not take more than 10 minutes for each su ...
+For the PDF invoice of your supplier that don't have an embedded XML file, you will have to create a `template file <https://github.com/invoice-x/invoice2data/blob/master/invoice2data/templates>`_ in YAML format in the invoice2data Python library. It is quite easy to do ; if you are familiar with `regexp <https://docs.python.org/2/library/re.html>`_, it should not take more than 10 minutes for ea ...
```
The link is now this one: https://github.com/invoice-x/invoice2data/tree/master/src/invoice2data/extract/templates

alexis-via commented on account_invoice_import_invoice2data/README.rst:
```diff
@@ -60,11 +60,11 @@ Go to the form view of the supplier and configure it with the following paramete
 Here are some hints to help you add a template for your supplier:
 
-* Take `Free SAS template file <https://github.com/m3nu/invoice2data/blob/master/invoice2data/templates/fr/fr.free.adsl-fiber.yml>`_ as an example. You will find a sample PDF invoice for this supplier under invoice2data/test/pdfs/2015-07-02-invoice_free_fiber.pdf
+* Take `Free SAS template file <https://github.com/invoice-x/invoice2data/blob/master/invoice2data/templates/fr/fr.free.adsl-fiber.yml>`_ as an example. You will find a sample PDF invoice for this supplier under invoice2data/test/pdfs/2015-07-02-invoice_free_fiber.pdf
```
The link is now this one: https://github.com/invoice-x/invoice2data/blob/master/src/invoice2data/extract/templates/fr/fr.free.adsl-fiber.yml

matteotrubini committed:
[10.0] Update invoice2data library reference link

From https://github.com/m3nu/invoice2data to https://github.com/invoice-x/invoice2data
